scripts/generate_graph.py

```
@@ -0,0 +1,117 @@
+
+import os
+import pickle
+import SimpleITK as sitk
```

Ruff (0.11.9)

`SimpleITK` imported but unused

Remove unused import: `SimpleITK`

(F401)